src/Admin/SupportLink.php

```diff
@@ -112,8 +110,8 @@ public function amp_validated_url_status_actions( $actions, WP_Post $post ) {
 		}
 
 		$query_args = [
-			'page'    => 'amp-support',
-			'post_id' => $post->ID,
+			'page' => 'amp-support',
+			'url'  => rawurlencode( $post->post_title ),
```

Comment:
@dhaval-parekh This will not generate post url:

![Screenshot 2021-11-22 at 10 34 31](https://user-images.githubusercontent.com/4476915/142837351-3d85b24c-dabf-4017-a518-00f545fb0560.png)

Probably something like this can be used instead:

```suggestion
			'url'  => get_permalink( $post ),
```

Reply:
@bartoszgadomski That would be great if you provide steps to reproduce.

We need to use `$post->post_title` since here we need a URL that is validated instead of the URL of that post itself. And we are storing validated URL as post title  (in `amp_validated_url` post type)

Reply:
I see... I tried to broke your code and looks like I went too far with that. As this piece of code is dedicated for `amp_validated_url` post type only, I think we are ok 👍